examples/sha256/src/sha256.rs

```diff
@@ -10,7 +10,7 @@ bytes!(Block, BLOCK_SIZE);
 array!(OpTableType, 12, usize);
 bytes!(Sha256Digest, HASH_SIZE);
 array!(RoundConstantsTable, K_SIZE, U32);
-array!(Hash, 8, U32);
+array!(HashType, 8, U32);
```

Comment:
I presume you ran into conflicts with the `hash` function?
The F* backend solved the general issue of colliding symbols by prepending a `_t` to types.
https://github.com/hacspec/hacspec/blob/8330fdd906ef9ce280c2291377ca2f6e24323f06/fstar/Hacspec.Sha256.fst#L25
Could this be an option for the coq backend as well to solve the issue in general and not only for this spec?

Reply:
Yes, I think something like that would work. It might be worthwhile to go through and see if anything else would benefit from appending some symbol.

Reply:
I fixed this in a pull request to `Kasserne:master` and reverted the changes to Sha256.